src/profileData/luisjimenez19.json

```diff
@@ -0,0 +1,10 @@
+{
+    "username": "LuisJimenez19",
+    "name": "Luis Ángel Jimenez",
+    "image": "https://github.com/Kiran1689/Awesome-Dev-Portfolios/assets/102745510/5e94fd47-c62f-42c1-8fa3-91072d5a1437",
```

Comment:
Resize your image, the image resolution should be **1900 x 900 (Width x Height)**